frontend/src/views/Setup.vue

```diff
@@ -0,0 +1,228 @@
+<script setup lang="ts">
+import router from "@/plugins/router";
+import userApi from "@/services/api/user";
+import storeHeartbeat from "@/stores/heartbeat";
+import type { Events } from "@/types/emitter";
+import type { Emitter } from "mitt";
+import { useRoute } from "vue-router";
+import { computed, inject, ref } from "vue";
+import { useDisplay } from "vuetify";
+
+// Props
+const { xs, smAndDown } = useDisplay();
+const emitter = inject<Emitter<Events>>("emitter");
+const heartbeat = storeHeartbeat();
+const route = useRoute()
+const visiblePassword = ref(false);
+// Use a computed property to reactively update metadataOptions based on heartbeat
+const metadataOptions = computed(() => [
+  {
+    name: "IGDB",
+    value: "igdb",
+    logo_path: "/assets/scrappers/igdb.png",
+    disabled: !heartbeat.value.METADATA_SOURCES?.IGDB_API_ENABLED,
+  },
+  {
+    name: "MobyGames",
+    value: "moby",
+    logo_path: "/assets/scrappers/moby.png",
+    disabled: !heartbeat.value.METADATA_SOURCES?.MOBY_API_ENABLED,
+  },
+  {
+    name: "SteamgridDB",
+    value: "sgdb",
+    logo_path: "/assets/scrappers/sgdb.png",
+    disabled: !heartbeat.value.STEAMGRIDDB_ENABLED,
```

Comment:
should also be under `METADATA_SOURCES`

```suggestion
    disabled: !heartbeat.value.METADATA_SOURCES?.STEAMGRIDDB_ENABLED,
```